EnvironTest.testEnviron fails with aiohttp 3.8.1

We've upgraded to aiohttp>=3.8.0 recently in nixpkgs and I noticed this package now fails the following test:

```
___________________________ EnvironTest.testEnviron ____________________________

self = <tests.test_environ.EnvironTest testMethod=testEnviron>

    def testEnviron(self) -> None:
        with self.run_server(assert_environ) as client:
>           client.assert_response(headers={
                "Foo": "bar",
            })

tests/test_environ.py:99: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/base.py:43: in assert_response
    self._test_case.assertEqual(response.status, 200)
E   AssertionError: 500 != 200
------------------------------ Captured log call -------------------------------
ERROR    aiohttp.server:web_protocol.py:405 Error handling request
Traceback (most recent call last):
  File "/nix/store/cpi0v7m6l2y47banahzihgpca47diryv-python3.9-aiohttp-3.8.1/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/nix/store/cpi0v7m6l2y47banahzihgpca47diryv-python3.9-aiohttp-3.8.1/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/build/source/aiohttp_wsgi/wsgi.py", line 294, in handle_request
    return await self._loop.run_in_executor(
  File "/nix/store/nffa4bgyi43wcrw3g3z0cmwy9zzz528y-python3-3.9.6/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/build/source/aiohttp_wsgi/wsgi.py", line 147, in _run_application
    body_iterable = application(environ, start_response)
  File "/build/source/tests/test_environ.py", line 13, in do_environ_application
    func(environ)
  File "/build/source/tests/test_environ.py", line 23, in assert_environ
    assert environ["CONTENT_TYPE"] == ""
AssertionError: assert 'application/octet-stream' == ''
  + application/octet-stream
```

--- a/tests/test_environ.py
+++ b/tests/test_environ.py
@@ -20,7 +20,7 @@ def assert_environ(environ: WSGIEnviron) -> None:
     assert environ["REQUEST_METHOD"] == "GET"
     assert environ["SCRIPT_NAME"] == ""
     assert environ["PATH_INFO"] == "/"
-    assert environ["CONTENT_TYPE"] == ""
+    assert environ["CONTENT_TYPE"] == "application/octet-stream"
     assert environ["CONTENT_LENGTH"] == "0"
     assert environ["SERVER_NAME"] == "127.0.0.1"
     assert int(environ["SERVER_PORT"]) > 0
@@ -97,6 +97,7 @@ class EnvironTest(AsyncTestCase):
     def testEnviron(self) -> None:
         with self.run_server(assert_environ) as client:
             client.assert_response(headers={
+                "Content-Type": "application/octet-stream",
                 "Foo": "bar",
             })
 

